ios/RNSScreenStackHeaderConfig.m

```diff
@@ -355,6 +355,21 @@ + (void)updateViewController:(UIViewController *)vc withConfig:(RNSScreenStackHe
   if (shouldHide) {
     return;
   }
+  
+  // if we don't explicitly change the direction in config, we should get it from the phone settings
+  if (config.direction == UISemanticContentAttributeUnspecified) {
+    if ([NSLocale preferredLanguages].count != 0 &&
```

Comment:
I think if this isn't the default behavior on the OS (I mean using the system settings by default), we shouldn't be following this approach either. If we want to do this anyways, it should work the same way on Android

Reply:
On Android, it works without the need of setting anything despite `android:supportsRtl="true"` in the manifest.